optaplanner-persistence/optaplanner-persistence-jaxb/src/test/java/org/optaplanner/persistence/jaxb/api/score/AbstractScoreJaxbAdapterTest.java

```diff
@@ -65,7 +65,7 @@ public abstract class AbstractScoreJaxbAdapterTest {
     protected <Score_ extends Score<Score_>, W extends TestScoreWrapper<Score_>> void assertSerializeAndDeserializeJson(
             Score_ expectedScore,
             W input) {
-        System.setProperty("javax.xml.bind.context.factory", "org.eclipse.persistence.jaxb.JAXBContextFactory");
+        System.setProperty("jakarta.xml.bind.JAXBContextFactory", "org.eclipse.persistence.jaxb.JAXBContextFactory");
```

Comment:
I think we'll be able to use an if switch, if we do a classpath check for either a Jakarta/Javax class. That way, one less problem for the migration script.